src/pages/edge/lifecycle-for-edge-network/behavior-reference.md

```diff
@@ -0,0 +1,376 @@
+    }
+  }
+}%%
+graph LR
+    graph1 ==> graph2
+
+    subgraph graph1 [" "]
+        direction TB
+        B(1a.<br/><code>lifecycleStart</code>) ==> C("1b.<br/>Application Launch (Foreground)")
+        class graph1 transparentSubgraph;
+    end
+
+    subgraph graph2 [" "]
+        direction TB
+        E(2a.<br/><code>lifecycleStart</code>) ==> F("2b.<br/>Call ignored, no event dispatched")
+        class graph2 transparentSubgraph;
+    end
+
+    classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
+    classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
+    classDef transparentSubgraph fill:transparent,stroke:#7a7a7a;
+
+    class B,C,F regularBox;
+    class E incorrectBox;
+ -->
+
+#### Consecutive `lifecyclePause` API calls
+
+![](./assets/index/lifecycle-pause-after-pause.svg)
+
+<!-- 
+%%{
+  init: {
+    'themeVariables': {
+      'lineColor': '#7a7a7a'
+    }
+  }
+}%%
+graph LR
+    graph1 ==> graph2
+
+    subgraph graph1 [" "]
+        direction TB
+        B(1a.<br/><code>lifecyclePause</code>) ==> C("1b.<br/>Application Close (Background)")
+        class graph1 transparentSubgraph;
+    end
+
+    subgraph graph2 [" "]
+        direction TB
+        E(2a.<br/><code>lifecyclePause</code>) ==> F("2b.<br/>Call ignored, no event dispatched")
+        class graph2 transparentSubgraph;
+    end
+
+    classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
+    classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
+    classDef transparentSubgraph fill:transparent,stroke:#7a7a7a;
+
+    class B,C,F regularBox;
+    class E incorrectBox;
+ -->
+
+## Expected Lifecycle scenarios
+
+The following Lifecycle workflows show examples of the expected ordering of Lifecycle events.
+
+### App launch and close
+
+This scenario shows the standard Lifecycle workflow, with expected Lifecycle API calls.
+
+![](./assets/index/lifecycle-app-launch-and-close.svg)
+
+<!-- 
+%%{
+  init: {
+    'themeVariables': {
+      'lineColor': '#7a7a7a'
+    }
+  }
+}%%
+graph LR
+    A(["1.<br/>App launched"]) ==> graph1
+    graph1 ==> D(["3.<br/>App closed by user"])
+    D ==> graph2
+
+    subgraph graph1 [" "]
+        direction TB
+        B("2a.<br/><code>lifecycleStart</code><br/>(Start of new session)") ==> C("2b.<br/>Application Launch (Foreground)")
+    end
+
+    subgraph graph2 [" "]
+        direction TB
+        E(4a.<br/><code>lifecyclePause</code>) ==> F("4b.<br/>Application Close (Background)")
+    end
+
+    classDef dashedPill fill:#d3d3d3,stroke:#000,stroke-dasharray: 5 5,color:#000;
+    classDef regularPill fill:#3273de,stroke:#3273de,color:#fff;
+    classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
+    classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
+    classDef transparentSubgraph fill:transparent,stroke:#7a7a7a;
+
+    class A,D regularPill;
+    class B,C,E,F regularBox;
+    class graph1,graph2 transparentSubgraph;
+-->
+
+### On app crash or force close
+
```

Comment:
Is this scenario same as the above Consecutive 1a. lifecycleStart, 2a lifecycleStart?
Or the consecutive lifecycleStart in the above has to be happened in the same session without a forced closed?